libraries/chain/include/steem/chain/util/smt_token.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#ifdef STEEM_ENABLE_SMT
+
+#include <steem/chain/database.hpp>
+#include <steem/chain/smt_objects/smt_token_object.hpp>
+#include <steem/protocol/asset_symbol.hpp>
+
+namespace steem { namespace chain { namespace util {
+
+const smt_token_object* smt_token_lookup( database& db, uint32_t nai );
```

Comment:
Change these to `find_smt_token` to match naming convention of other database helper methods.